geode-docs/tools_modules/geode_for_redis.html.md.erb

```diff
@@ -229,27 +233,12 @@ integer (+/-&nbsp;9223372036854775807) for CURSOR.
 | server       | redis_version<br/>redis_mode (always returns "cluster" because <%=vars.product_name%> for Redis always runs in cluster mode.)<br/>tcp_port<br/>uptime_in_seconds<br/>uptime_in_days |
 | stats        | total_commands_processed<br/>instantaneous_ops_per_sec<br/>total_net_input_bytes<br/>instantaneous_input_kbps<br/>total_connections_received<br/>keyspace_hits<br/>keyspace_misses<br/>evicted_keys (always returns 0)<br/>rejected_connections (always returns 0)<br/>pubsub_channels<br/>pubsub_patterns|
 
-## <a id="advantages-over-redis"></a>Advantages of <%=vars.product_name%> over Redis
```

Comment:
Part of the original ticket was to add a section for advantages of using Geode for Redis, but if you don't think that it belongs here then ok

Reply:
I removed this section as it did not feel as though it was the best place or way to do compare this implementation vs OSS Redis.